dedupe/labeler.py

```diff
@@ -390,12 +383,11 @@ def mark(self, pairs: TrainingExamples, y: LabelsLike) -> None:
         self.y = numpy.concatenate([self.y, y])  # type: ignore[arg-type]
         self.pairs.extend(pairs)
 
-        self.fit_transform(self.pairs, self.y)
-
-    def fit_transform(self, pairs: TrainingExamples, y: LabelsLike) -> None:
+        self.fit(self.pairs, self.y)
 
+    def fit(self, pairs: TrainingExamples, y: LabelsLike) -> None:
         for learner in self.learners:
-            learner.fit_transform(pairs, y)
+            learner.fit(pairs, y)
```

Comment:
`fit` is a bad verb for something that both sets the data on a learner and retrains the learner. I think maybe we want to get rid of fit from the public API and have `mark` be responsible for all of this.

Reply:
See new version and comment above.